python/ray/experimental/state/common.py

```diff
@@ -564,6 +564,8 @@ class TaskState(StateSchema):
     start_time_ms: Optional[int] = state_column(detail=True, filterable=False)
     #: The time when the task is finished or failed. A Unix timestamp in ms.
     end_time_ms: Optional[int] = state_column(detail=True, filterable=False)
+    #: Task error type.
+    error_type: Optional[str] = state_column(detail=False, filterable=False)
```

Comment:
We need exit_detail here to actually store the exceptions

Reply:
could be in a follow up PR

Reply:
Sure, but the error info has a rather weird structure (with `oneof`) so needs some logic parsing the actual exit detail so I was leaving it out.